src/frontend/js/widgets/LanguageSelector/index.spec.tsx

```diff
@@ -37,17 +37,16 @@ describe('<LanguageSelector />', () => {
       selector: 'button',
     });
 
-    await userEvent.click(button);
+    await act(async () => {
+      await userEvent.click(button);
```

Comment:
Wasn't `userEvent` supposed to execute a `act` itself? Anyway these `act` stories are really weird. 😅